igor.dzetcyna/1/lesson_2/task1-class_Pet.rb

```diff
@@ -0,0 +1,112 @@
+class Pet
+  attr_accessor :name, :health, :eat, :play, :sleep
+ 
+   def initialize (name, health, eat, play, sleep)
+     @name = name
+     @health = health  
+     @play = eat
+     @sleep = sleep 
+     @eat = play
+
+     print "Enter the name of the pet: "
+     name = gets.chomp().to_s
+     pet = Pet.new(name, 100, 100, 100, 100)
```

Comment:
you call new instance from initialize method of this class, it's looks like recursion